File: src/main/java/com/examples/withimports/Question.java
package com.examples.withimports;

import edu.illinois.cs.cs125.questioner.lib.Correct;
import java.util.Arrays;
import java.util.List;

/*
 * Example to test use of import statements.
 */

@Correct(name = "With Imports", version = "2022.7.0", author = "challen@illinois.edu")
public class Question {
  public static List<Integer> max(int first, int second) {
    return Arrays.asList(first, second);
  }
}

File: src/main/java/com/examples/withimports/correct/kotlin/Question.kt
@file:AlsoCorrect

package com.examples.withimports.correct.kotlin

import edu.illinois.cs.cs125.questioner.lib.AlsoCorrect
import java.util.Arrays

/*
 * Test Kotlin top-level method with import statements.
 */

@Suppress("ReplaceJavaStaticMethodWithKotlinAnalog")
fun max(first: Int, second: Int): List<Int> {
  return Arrays.asList(first, second)
}